wallet/src/extkey.rs

```diff
@@ -202,43 +205,30 @@ impl ExtendedKey {
 		Ok(ext_key)
 	}
 
-	/// Return the identifier of the key, which is the
-	/// Hash160 of the private key
+	/// Return the identifier of the key
+	/// which is the blake2b hash (20 bit digest)
 	pub fn identifier(&self) -> Identifier {
-		let mut sha = Sha256::new();
-		sha.input(&self.key[..]);
-
-		let mut shres = [0; 32];
-		sha.result(&mut shres);
-
-		let mut ripe = Ripemd160::new();
-		ripe.input(&shres[..]);
-
-		let mut identifier = [0; 20];
-		ripe.result(&mut identifier);
-		Identifier::from_bytes(&identifier)
+		let identifier = blake2b(20, &[], &self.key[..]);
```

Comment:
Do we want to stick with `20` here to match original ripemd160 impl?

Reply:
I think so, seems like a sane default.